translations/en-us.yml

```diff
@@ -95,13 +95,33 @@ general:
         preprints: Preprints
         registries: Registries
     other: Other
+    checkout: Checkout
+    metadata: Metadata
+    file_description: Description
+    type: Type
+    funding_agency: 'Funding Agency'
+    type_select: 'File Type Select'
+    example_funder: 'Funder One'
+    journal_article: 'Journal Article'
+    audio_video: 'Audio/Video'
+    dataset: 'Dataset'
+    image: 'Image'
+    model: 'Model'
+    software: 'Software'
+    book: 'Book'
+    poster: 'Poster'
+    presentation: 'Presentation'
+    tags: 'Tags'
+    science: 'science'
+    engineering: 'engineering'
```

Comment:
All these metadata-related translation strings can go away.

Reply:
Ok, I can move the tag relevant strings for when we work on ENG-3534: File detail: Tags